{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-9918", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": [{"vendor_name": "qpdf_project", "product": {"product_data": [{"product_name": "qpdf", "version": {"version_data": [{"version_value": "8.0.2", "version_affected": "<="}]}}]}}, {"vendor_name": "canonical", "product": {"product_data": [{"product_name": "ubuntu_linux", "version": {"version_data": [{"version_value": "14.04", "version_affected": "="}, {"version_value": "16.04", "version_affected": "="}, {"version_value": "17.10", "version_affected": "="}]}}]}}]}}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-399"}]}]}, "references": {"reference_data": [{"url": "https://github.com/qpdf/qpdf/commit/b4d6cf6836ce025ba1811b7bbec52680c7204223", "name": "https://github.com/qpdf/qpdf/commit/b4d6cf6836ce025ba1811b7bbec52680c7204223", "refsource": "MISC", "tags": ["Patch"]}, {"url": "https://github.com/qpdf/qpdf/issues/202", "name": "https://github.com/qpdf/qpdf/issues/202", "refsource": "MISC", "tags": ["Exploit", "Issue Tracking", "Patch", "Third Party Advisory"]}, {"url": "https://usn.ubuntu.com/3638-1/", "name": "USN-3638-1", "refsource": "UBUNTU", "tags": ["Third Party Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": "libqpdf.a in QPDF through 8.0.2 mishandles certain 'expected dictionary key but found non-name object' cases, allowing remote attackers to cause a denial of service (stack exhaustion), related to the QPDFObjectHandle and QPDF_Dictionary classes, because nesting in direct objects is not restricted."}]}}
